taxonomy/algolia/serializers.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+Serializers to convert db data to a dictionary that algolia can understand.
+
+This module depends on serializers provided by django-rest-framework.
+"""
+
+from rest_framework import serializers
+
+from ..models import Job, JobPostings, JobSkills
```

Comment:
what if we import like `from taxonomy.models import Job, JobPostings, JobSkills`?